src/controllers/tasks/people.with.significant.control.controller.ts

```diff
@@ -118,8 +118,18 @@ const updateCsSubmission = (currentCsSubmission: ConfirmationStatementSubmission
 
 const getPscTypeTemplate = (pscAppointmentType: string): string => {
   switch (pscAppointmentType) {
-      case appointmentTypes.INDIVIDUAL_PSC: return "psc";
-      case appointmentTypes.RLE_PSC: return "rle";
+      case appointmentTypes.INDIVIDUAL_PSC: return appointmentTypeNames.PSC;
+      case appointmentTypes.RLE_PSC: return appointmentTypeNames.RLE;
       default: throw createAndLogError(`Unknown PSC type: ${pscAppointmentType}`);
   }
 };
+
+const handleDateOfBirth = (pscAppointmentType: string, psc: PersonOfSignificantControl): string => {
+  if (pscAppointmentType === appointmentTypeNames.RLE) {
+    return "";
+  }
+  if (psc.dateOfBirth) {
+    return toReadableFormatMonthYear(psc.dateOfBirth.month, psc.dateOfBirth.year);
+  }
+  throw createAndLogError(`Date of birth missing for individual psc ${psc.nameElements.surname}`);
```

Comment:
can you add an undefined check to ${psc.nameElements.surname} like so ${psc.nameElements?.surname}
this will return undefined if nameElements is undefined. Otherwise it will throw a null pointer when trying to access surname on undefined and will fail to log the message. With the'?' in place, the message should display but with 'undefined' in the message placeholder.